homework/05/ChatSys/src/Client.java

```diff
@@ -0,0 +1,74 @@
+import com.sun.source.util.SourcePositions;
+
+import java.io.*;
+import java.net.Socket;
+
+public class Client implements Runnable{
+
+    private Socket client;
+    private BufferedReader in;
+    private PrintWriter out;
+    private boolean done;
+
+    @Override
+    public void run() {
+        try{
+            client = new Socket("127.0.0.1", 4444);

```

Comment:
You have given predefined values of the Server IP and port number. They should be given by the user by using the "/connect" command, which is missing